### ISSUE TITLE
[5.2] Setting `pageName` property for Table\Grid

### ISSUE THREAD

**Issue by stevebauman** [orchestral/html#51]
Is it possible to set the `pageName` property on table grids? Setting it regularly results in an:

```php
InvalidArgumentException in Orchestra\Html\Grid.php line 250:
Require values to be an array.
```

And setting it as an array doesn't take effect, or using the `set('pageName', 'page');` method.

The `pageName` property is set to `protected`, so maybe it could be removed and be apart of grid's attributes, have a setter (`$table->pageName('page')`), or be `public` instead?

Please let me know if there's a different solution, thanks!

**Comment by crynobone** [comment]
How did you set it? It is accessible via __set, __get and __call

**Comment by crynobone** [comment]
I see what you mean now.

**Comment by stevebauman** [comment]
> How did you set it? It is accessible via __set, __get and __call

> I see what you mean now.

Yea setting it regularly results in the exception:

```php
$table->pageName = 'new-page-name';
```

Is there any workaround atm other than overriding the current grid class?

**Comment by crynobone** [comment]
Fixed via 5578915

**Comment by stevebauman** [comment]
Awesome thanks a ton!
